lib/utils/__tests__/regexes.test.mjs

```diff
@@ -1,4 +1,6 @@
-import { atRuleRegexes } from '../regexes.mjs';
+import { describe, it } from '@jest/globals';
```

Comment:
nit

These imports are not needed in `*.test.mjs` files.

Reply:
Thanks! I've fixed it.
[Remover unnecessary import](https://github.com/stylelint/stylelint/pull/8331/commits/79128145c76329ca763b12b9e82057675ba02d95)